migration/migrating_3_4/about-migration.adoc

```diff
@@ -16,4 +16,3 @@ xref:../../migration/migrating_3_4/migrating-application-workloads-3-4.adoc#migr
 Learn about and use the tools to perform your migration:
```

Comment:
@apinnick I apologize, I didn't catch that this intro sentence needs to be updated because there is only one tool to learn about and use because the CPMA has been removed. I suggest changing this to a single sentence and remove the list item:

> Learn about and use the Migration Toolkit for Containers (MTC) to migrate your application workloads.

Reply:
No problem.

Reply:
Please let me know when you have a new PR and I can quickly review again and merge/CP. Sorry again for not catching this the first time around.